cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create two migration plan with same name
+        expectedResults:
+            1.
+            2. Duplicate migration plan name should not be allowed.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_vm_name_with_special_characters_can_be_imported():
+    """
+    title: OSP: Test if VM name with special characters can be imported (It
+    should allow such imports)
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: low
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan and import vm name containing special chars
+        expectedResults:
+            1.
+            2. Importing Vm with special chars in name should be allowed.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_if_no_password_is_exposed_in_logs_during_migration():
+    """
+    OSP: Test if no password is exposed in logs during migration
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. logs should not show password during migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_associated_tags_before_and_after_migration_department_accounting_kind():
+    """
+    title : OSP: Test associated tags before and after migration
+    (Ex: Department:Accounting)
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan with a VM tagged to some category
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Migrated Vm should have the same tag associated.
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_vmware65_vm_with_multiple_nics():
+    """
+    title : OSP: vmware65-Test VM with multiple NICs with single IP (IPv6 to first
+    NIC and IPv4 to second)
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan with VM with multiple nics
```

Comment:
Can you please elaborate on how should be NICs setup here. I vaguely remember this but not entirely sure on how you test `IPv6 to first NIC and IPv4 to second)`